source/js/components/join/join.scss

```diff
@@ -78,6 +78,9 @@
   }
 
   &.full-page {
+    margin-top: 2rem;
+    margin-bottom: 3rem;
+
```

Comment:
Had to add custom css here since bootstrap doesn't have a class for margin `2rem`.  I can turn `margin-bottom: 3rem` into using `.mb-5` on the element if preferred.